modules/analysis/rosbag-image-pipeline/deployspec.yaml

```diff
@@ -0,0 +1,45 @@
+deploy:
+  phases:
+    install:
+      commands:
+      - npm install -g aws-cdk@2.20.0
+      - pip install -r requirements.txt
+    build:
+      commands:
+      - cdk deploy --require-approval never --progress events --app "python app.py" --outputs-file ./cdk-exports.json
+      # Here we export some env vars
+      - export ADDF_MODULE_METADATA=$(python -c "import json; file=open('cdk-exports.json'); print(json.load(file)['addf-${ADDF_DEPLOYMENT_NAME}-${ADDF_MODULE_NAME}']['metadata'])")
+      - export ECR_REPO_NAME=$(echo ${ADDF_MODULE_METADATA} | jq -r ".EcrRepoName")
+      - export REPOSITORY_URI=$AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$ECR_REPO_NAME
+      # - Here we write env values into the dag_config.py file for use by the DAGs
+      - export DAG_DIR="image_dags"
```

Comment:
Is this an env var being brought from manifest file or just hardcoded here?

Reply:
This is just a hardcoding of the name of the local directory containing the dag definitions.